feature/bgp/policybase/otg_tests/link_bandwidth_test/link_bandwidth_test.go

```diff
@@ -229,32 +259,64 @@ func TestBGPLinkBandwidth(t *testing.T) {
-	var extCommunityEnableCLIConfig string
-	switch dut.Vendor() {
-	case ondatra.CISCO:
-		extCommunityEnableCLIConfig = fmt.Sprintf("router bgp %v instance BGP neighbor-group %v \n ebgp-recv-extcommunity-dmz \n ebgp-send-extcommunity-dmz\n", dutAS, cfgplugins.BGPPeerGroup1)
-	default:
-		t.Fatalf("Unsupported vendor %s for deviation 'BgpExplicitExtendedCommunityEnable'", dut.Vendor())
-	}
-	helpers.GnmiCLIConfig(t, dut, extCommunityEnableCLIConfig)
+	if deviations.BgpExplicitExtendedCommunityEnable(dut) {
+		var extCommunityEnableCLIConfig string
+		switch dut.Vendor() {
+		case ondatra.CISCO:
+			extCommunityEnableCLIConfig = fmt.Sprintf("router bgp %v instance BGP neighbor-group %v \n ebgp-recv-extcommunity-dmz \n ebgp-send-extcommunity-dmz\n", dutAS, cfgplugins.BGPPeerGroup1)
+		default:
+			t.Fatalf("Unsupported vendor %s for deviation 'BgpExplicitExtendedCommunityEnable'", dut.Vendor())
+		}
+		helpers.GnmiCLIConfig(t, dut, extCommunityEnableCLIConfig)
+	}
 }
 
-func applyPolicyDut(t *testing.T, dut *ondatra.DUTDevice, policyName string) {
-	// Apply ipv4 policy to bgp neighbour.
+func applyImportPolicyDut(t *testing.T, dut *ondatra.DUTDevice, policyName string) {
 	root := &oc.Root{}
 	dni := deviations.DefaultNetworkInstance(dut)
-	path := gnmi.OC().NetworkInstance(dni).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, bgpName).
-		Bgp().Neighbor(atePort1.IPv4).AfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).ApplyPolicy()
 
-	policy := root.GetOrCreateNetworkInstance(dni).GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, bgpName).
-		GetOrCreateBgp().GetOrCreateNeighbor(atePort1.IPv4).GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).GetOrCreateApplyPolicy()
+	// Apply ipv4 policy to bgp neighbour.
+	path := gnmi.OC().NetworkInstance(dni).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, bgpName).Bgp().Neighbor(atePort1.IPv4).AfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).ApplyPolicy()
+	policy := root.GetOrCreateNetworkInstance(dni).GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, bgpName).GetOrCreateBgp().GetOrCreateNeighbor(atePort1.IPv4).GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV4_UNICAST).GetOrCreateApplyPolicy()
 	policy.SetImportPolicy([]string{policyName})
+	gnmi.Delete(t, dut, path.Config())
 	gnmi.Replace(t, dut, path.Config(), policy)
 
 	// Apply ipv6 policy to bgp neighbour.
 	path = gnmi.OC().NetworkInstance(dni).Protocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, bgpName).Bgp().Neighbor(atePort1.IPv6).AfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST).ApplyPolicy()
 	policy = root.GetOrCreateNetworkInstance(dni).GetOrCreateProtocol(oc.PolicyTypes_INSTALL_PROTOCOL_TYPE_BGP, bgpName).GetOrCreateBgp().GetOrCreateNeighbor(atePort1.IPv6).GetOrCreateAfiSafi(oc.BgpTypes_AFI_SAFI_TYPE_IPV6_UNICAST).GetOrCreateApplyPolicy()
 	policy.SetImportPolicy([]string{policyName})
+	gnmi.Delete(t, dut, path.Config())
+	gnmi.Replace(t, dut, path.Config(), policy)
+
```

Comment:
Need to delete leftover export-policy from previous case on port1 to avoid mixup & failing verification in both v4 & v6.
1. `ExportPolicy set not_match_100_set_linkbw_1M` is leftover on port1 while `ImportPolicy set match_100_set_linkbw_2G` is applied on port2.
2.  'ExportPolicy set match_100_set_linkbw_2G` is leftover on port1 while `ImportPolicy set del_linkbw` is applied on port2.